uniffi_bindgen/src/lib.rs

```diff
@@ -113,7 +113,7 @@ pub mod interface;
 pub mod scaffolding;
 
 use bindings::TargetLanguage;
-use interface::ComponentInterface;
+pub use interface::ComponentInterface;
```

Comment:
Do we need this? I have no objections, but it looks like all of `interface` is pub on line 112 - not that it matters much, there is convenience in getting access to the `ComponentInterface` with one less module.. maybe `interface` should be a private module and only expose `ComponentInterface` 🤷

Reply:
I'm never completely sure how to handle this.  In the bindings code we use a several other components from `interface`.  Maybe we could make that module private and add `pub use interface::*`.  Now that these are all separate crates, it seems more natural to import from the `uniffi_bindgen` rather `uniffi_bindgen::interface`.